rqd/rqd/__main__.py

```diff
@@ -95,7 +95,8 @@ def usage():
     print("  -d | --daemon          => Run as daemon", file=s)
     print("       --nimbyoff        => Disables nimby activation", file=s)
     print("  -c                     => Provide an alternate config file", file=s)
-    print("                            Defaults to /etc/rqd3/rqd3.conf", file=s)
+    print("                            On Linux: defaults to /etc/opencue/rqd.conf", file=s)
+    print("                            On Windows: Defaults to %LOCALAPPDATA%/OpenCue/rqd.conf", file=s)
```

Comment:
This line is causing the lint check to fail as it's too long.

If there's a way to break it up over multiple lines, great. But since this line is intentionally long in order for the help text to be formatted properly, I'm also fine with disabling that pylint check for this section.

You can do that like:

```
# pylint: disable=line-too-long
print("SYNOPSIS", file=s)
... other help text lines ... 
# pylint: enable=line-too-long
```

Reply:
I tried using string interpolation like `"On Linux: defaults to %s", linux_conf_path`, but then I get another PyLint warning (i.e. consider-using-f-string). But if I use f-string, then it creates an undesired space shift compared to the other lines.

So I opted for a different approach and used a multiline f-string instead (i.e. `f"""`), which is pretty common IMO for usage strings.

Here's the output:
```text
SYNOPSIS
  D:\dev\ubisoft\opencue\venv\Scripts\rqd [options]

  -d | --daemon          => Run as daemon
       --nimbyoff        => Disables nimby activation
  -c                     => Provide an alternate config file
                            On Linux: defaults to /etc/opencue/rqd.conf
                            On Windows: Defaults to %LOCALAPPDATA%/OpenCue/rqd.conf
                            Config file is optional
```